src/FqnUtilities.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace LanguageServer;
+
+use phpDocumentor\Reflection\{Type, Types};
+use Microsoft\PhpParser;
+
+class FqnUtilities
+{
+    /**
+     * Returns all possible FQNs in a type
+     *
+     * @param Type|null $type
+     * @return string[]
+     */
+    public static function getFqnsFromType($type): array
```

Comment:
Sure, you could also refer them by their namespace just like you would refer to a static class, e.g.
```php
namespace LanguageServer\FqnUtilities;
function foo() {}
```
```php
namespace LanguageServer;
FqnUtilities\foo();
```
Aliasing them is just for being able to omit the prefix (if the function name is clear enough).

The file just needs to be added to composer.jsons `autoload.files` array.